### DISCUSSION TITLE
First Set of Outcomes

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#51] [General]
We have a subgroup that is working on an initial outcome (Visible keyboard focus) but when they are done, we'd like to set up 5-6 subgroups to work on a first set of outcomes. 

Based on discussion in the [6 February Meeting](https://www.w3.org/2024/02/06-ag-minutes.html#t02), the group should include a diverse, representative set of outcomes that includes a mix of guidance covered in WCAG 2.2 and gaps in WCAG 2.2. 

Please propose and vote on a set of outcomes for the first set of subgroups that meets these criteria:
- thumbs up for support, 
- thumbs down for concerns, 
- eyes for neutral or watching) 

Please note: This thread is to select the set of outcomes for work and not to start working on those outcomes.

**Comment by rachaelbradley** [comment]
(Chair hat off) I'm suggesting the following set because, along with Visible keyboard Focus, I think it will give us a mix of old and new; qualitative tests, quantitative tests, and assertions; discussion around accessibility supported, and guidance across various categories and user needs.

1. **Section Labels:** Sections of content have clear visual and programmatic labeled
2. **Implied meaning in figurative language**: Explain figurative language or non-literal language such as jokes, sarcasm, hyperbole, metaphors, similes, and idioms.
3. **Adjust Color**: Users are able to adjust color schemes 
4. **Haptic stimulation**: Protects users from haptic over-stimulation 
5. **Keyboard Only**: All functionality can be performed through the keyboard interface only, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

**Reply by alastc** [reply]
Perhaps as part of this we could split the Section Labels guideline? Once you break down the methods, there are some which don't apply visually, and some which don't apply to code.

Apart from that, it seems like a wide range for 5 outcomes!

**Comment by GreggVan** [comment]
+1 to @chaals comments — or at least his bottom line  (I had trouble following some of it) 

To make things clearer (hopefully — this is a topic that it is easy to get confused by) 

Keyboard interface 
- does not refer to a physical keyboard - it is the interface the software gets keystrokes from the OS
- it is critical to accessibility by many disabilities
	- and mousekeys is NOT an equivalent 
	- nor is voice or pointing or any other technique. (They are ALL good things for other PWD and for some people who migh use an AT that contrrols content through the keyboard interface - but they are not useable by all who need keyboard interface access. 

so 
— keyboard interface remains a critical and fundamental requirement for accessibility
— other things are good supplements but not substitutes for keyboard interface access
– mousekeys does not count as “operates with a keyboad interface” because it come to the program through the mouse interface, not the programs “keyboard interface” 

Best 

Gregg
